src/components/CreateAddress/AddressGenerator.jsx

```diff
@@ -94,21 +91,35 @@ class AddressGenerator extends React.Component {
     return multisig;
   };
 
-  downloadAddressDetails = (event) => {
-    event.preventDefault();
-    const multisig = this.generateMultisig();
-    const body = this.addressDetailsText(multisig);
-    const filename = this.addressDetailsFilename(multisig);
-    downloadFile(body, filename);
+  const addressDetailsFilename = (multisig) => {
+    return `bitcoin-${requiredSigners}-of-${totalSigners}-${addressType}-${multisig.address}.txt`;
   };
 
-  addressDetailsFilename = (multisig) => {
-    const { totalSigners, requiredSigners, addressType } = this.props;
-    return `bitcoin-${requiredSigners}-of-${totalSigners}-${addressType}-${multisig.address}.txt`;
+  const publicKeyImporterBIP32Path = (number) => {
+    const publicKeyImporter = publicKeyImporters[number];
+    const bip32Path =
+      publicKeyImporter.method === "text"
+        ? "Unknown (make sure you have written this down previously!)"
+        : publicKeyImporter.bip32Path;
+    return `  * ${publicKeyImporter.name}: ${bip32Path}: ${publicKeyImporter.publicKey}`;
+  };
+
+  // @winsby: ?? duplicate variable names
+  const publicKeyImporterBIP32Paths = () => {
+    const publicKeyImporterBIP32Paths = [];
```

Comment:
lines 108 and 109 both say `const publicKeyImporterBIP32Paths ... ` one is a func and one is an array.